.github/workflows/dependency-review.yml

```diff
@@ -10,5 +10,39 @@ jobs:
     steps:
       - name: 'Checkout Repository'
         uses: actions/checkout@v4
-      - name: Dependency Review
+
+      - name: 'Dependency Review'
         uses: actions/dependency-review-action@main
+        id: review
+
+      - name: 'Check comment-content output'
+        if: always()
+        env:
+          COMMENT: ${{ steps.review.outputs.comment-content }}
+        run: |
+          test -n "$COMMENT"
+          echo "$COMMENT"
+
+      - name: 'Check dependency-changes output'
+        if: always()
+        env:
+          DEPENDENCY_CHANGES: ${{ steps.review.outputs.dependency-changes }}
+        run: echo "$DEPENDENCY_CHANGES" | jq
+
+      - name: 'Check vulnerable-changes output'
+        if: always()
+        env:
+          VULNERABLE_CHANGES: ${{ steps.review.outputs.vulnerable-changes }}
+        run: echo "$VULNERABLE_CHANGES" | jq
+
+      - name: 'Check invalid-license-changes output'
+        if: always()
+        env:
+          LICENSE_CHANGES: ${{ steps.review.outputs.invalid-license-changes }}
+        run: echo "$LICENSE_CHANGES" | jq
+
+      - name: 'Check denied-changes output'
+        if: always()
+        env:
+          DENIED_CHANGES: ${{ steps.review.outputs.denied-changes }}
+        run: echo "$DENIED_CHANGES" | jq
```

Comment:
I've been doing some testing with this branch of the Action, using an [output setup in my test repo](https://github.com/elireisman/monorepo/blob/elireisman/test-dr-ft/.github/workflows/dependency-review.yml) similar to what you have here, and I'm seeing some [unescaped entries in the output JSON](https://github.com/elireisman/monorepo/actions/runs/8285030033/job/22671857292)

Reply:
@febuiles I added it as test, not as a demo. But sure, I can remove it.

@elireisman there is an important difference between this code and your test:
- here the output data is stored in an env var, and then in the shell script it's just a reference to the var by its name, so the data is passed directly to `jq` (see example [logs here](https://github.com/laughedelic/dependency-review-action/actions/runs/8147727423/job/22269075026?pr=7#step:5:2))
- in your workflow you do a direct reference to `${{steps.dr_test.outputs.vulnerable-changes}}` which gets interpolated _before_ the shell script is evaluated (compare the [logs here](https://github.com/elireisman/monorepo/actions/runs/8271132492/job/22630211533#step:4:2))

You could probably wrap it in single quotes: `echo '${{steps.dr_test.outputs.vulnerable-changes}}'` but it might still break because of some quotes in the JSON body (e.g. `{"foo": "a'b"}` is a valid JSON, but will break that shell syntax).

I believe that the data in the output is stored correctly and doesn't need any extra escaping, but the usage need caution and the documentation for this could probably emphasize it.

Reply:
Thanks - can confirm, single quoted output still broke due to unescaped JSON values 👍 

I'm a bit concerned that the output is a bit delicate (or at least as you say requires a mention or example in the documentation?) if it requires a particular process to pass it along safely?

That said, this isn't my area of expertise 😆 so I'll defer to other reviewers if this is worth addressing or not. Thanks for the reply!